sources/visual/Bouncing.cpp

```diff
@@ -0,0 +1,28 @@
+#include "visual/Bouncing.h"
+
```

Comment:
Il faudrait mettre ce fichier en conformité avec l'indentation générale, qui est en espaces et non en tabulations.
Même remarque avec le fichier `Bouncing.h`.

Remarque : le projet contient une configuration d'indenteur (void `.clang-format` à la racine)
pouvant être utilisée avec le logiciel du même nom.

Reply:
bien noté, je vais essayer d'utiliser clang format